thrift0.12/build.gradle

```
@@ -1,15 +1,16 @@
-// Because 0.10, 0.11 and 0.12 are both ABI and API compatible with each other,
-// we do not publish 'armeria-thrift0.10' or 'armeria-thrift0.11' but just run
-// tests to make sure the classes generated by ':thrift' works with the classes
+// 0.10, 0.11, 0.12 and 0.13 are both ABI and API compatible with each other,
+// so we do not publish artifacts like 'armeria-thrift0.10' but just run tests
```

nit: `armeria-thrift0.12`?
```suggestion
// so we do not publish artifacts like 'armeria-thrift0.12' but just run tests
```

---

It's an example. We do not publish `armeria-thrift0.1[012]` and `armeria-thrift0.10` is just one of them.